vllm/entrypoints/openai/serving_chat.py

```diff
@@ -271,9 +271,14 @@ async def chat_completion_stream_generator(
                     # NOTE num_choices defaults to 1 so this usually executes
                     # once per request
                     for i in range(num_choices):
+
+                        # TODO - this breaks, it needs to include ALL fields
```

Comment:
What do you mean by "this breaks"? Is this WIP?

Reply:
Sorry that was an earlier annotation about what the source of the problem was, that I didn't clean up when I fixed it